### DISCUSSION TITLE
Move to Qt 6.2.1

### DISCUSSION THREAD

**Discussion by awesomeerictech** [emericg/WatchFlower#53] [General]
Hi EmericG when are you planning to move to Qt 6.2.1 LTS ?

**Comment by emericg** [comment]
Hi,
Long story short, I do not plan to move to Qt 6.2.1. Maybe 6.3, but probably 6.4 or later.

The problems with Qt6 are numerous, and there is no way to do a painless transition.
- If you have a simple app, or you're doing a new app, you should definitely start with Qt6.
- WatchFlower can run on Qt6 starting from Qt 6.2, but not on every platform. For instance it doesn't build on iOS. As you may have seen, I've started to support Qt6 in my code, but only when it's possible to keep support for Qt5.
- Most of my other applications however, cannot run at all on Qt 6.2, so if I start the transition process for some of the app, I'll have to maintain all of my shared code for both Qt5 and Qt6. There are still some missing Qt modules, and capabilities that just disappeared.
- With Qt6 I'll have to rework every CI buildbots, deployment scripts, every process and tool has changed...

The situation with Qt6 is clearly not ideal. Long term, it is necessary to switch.
But at the moment, it breaks nearly everything and yet does not bring any benefit.

**Reply by emericg** [reply]
Ok @awesomeerictech, the switch to Qt 6.3.0 (mandatory) is done! It's been a challenge, but everything seems to be running as it was with Qt 5. I'm sure a couple of bugs will pop up but I'll do a couple of beta release before the next big version.
